src/test/scala/group/scala/karazin/circe/literal/extras/arbitraries/ArbitrariesJavaDateTime.scala

```diff
@@ -0,0 +1,82 @@
+package group.scala.karazin.circe.literal.extras
+
+import cats.implicits._
+import io.circe.syntax._
+import io.circe.{Json, JsonNumber, JsonObject, parser}
+import org.scalacheck.Gen
+import org.scalacheck.Arbitrary
+import org.scalacheck.Prop._
+import io.circe.Encoder.encodeJsonObject
+import java.time._
+
+
+trait ArbitrariesJavaDateTime {
+  val instantGenerator: Gen[Instant] = Gen.choose(Instant.MIN, Instant.MAX)
+
+  val periodGenerator: Gen[Period] =
+    for {
+      d <- Gen.posNum[Int]
```

Comment:
Let's use a full word `day` instead.
It's not just a chicane, I saw a very good code with such a naming, and the code was unreadable. So let's keep the practise of well-named vals.